celery/schedules.py

```diff
@@ -428,14 +436,14 @@ def _expand_cronspec(cronspec, max_, min_=0):
         day of month or month of year.  The default is sufficient for minute,
         hour, and day of week.
         """
-        if isinstance(cronspec, numbers.Integral):
+        if isinstance(cronspec, int):
             result = {cronspec}
         elif isinstance(cronspec, str):
             result = crontab_parser(max_, min_).parse(cronspec)
         elif isinstance(cronspec, set):
             result = cronspec
         elif isinstance(cronspec, Iterable):
-            result = set(cronspec)
+            result = set(cronspec)  # type: ignore
```

Comment:
This line was especially problematic to `mpy` for some reason, so set it as ignore. For some reason it didn't want `cronspec` to be able to be a `str` here, even though you can convert a string to a set.